packages/react-components/src/Concepts/MigratingFromV8.stories.mdx

```diff
@@ -0,0 +1,182 @@
+
+Another change in version 9 is that we are collocating all our component examples in their respective package next to the component's code instead of having them in the separate `@fluentui/react-examples` package.
+
+Finally, other things to take note of are that version 9 does not include any "top-level`component files (i.e.`@fluentui/react/lib/Button`) nor does it have support for deep imports. The benefits of these decisions are that they allow us to not have a need for extra config in jest and other tools to remap from `esm`to`cjs`.
+
+---
+
+#### "You mentioned something about shorthands, what is that?"
+
+For version 9 we are utilizing a new composition model to create our components. We have decided to name the building blocks of these components **_slots_** which, as their name suggests, are slottable parts that we put together to build the component. Most of the things that `render[ComponentName]` function calls return are slots.
+
+**_Shorthands_** are the way in which we allow these _slot_ subparts within a component to be configurable. They allow a caller of the component to pass a variety of different inputs for a given _slot_, including but not limited to things like a string, JSX, an object or a function, depending on how the shorthand is configured. An example of this would be the `icon` shorthand in the `Button` component, with some uses exemplified below:
+
+```jsx
+<>
+  <Button icon={<JSXIcon />} />
+  <Button icon={{ children: <JSXIcon />, id: 'button-icon' }} />
+  <Button
+    icon={{
+      // wraps "icon" slot with a custom "div"
+      children: (Component, props) => (
+        <div>
+          <Component {...props} />
+        </div>
+      ),
+    }}
+  />
+</>
+```
+
+`resolveShorthandProps` and `getSlots` are some helpers that allow us to support and parse these dynamic _shorthand_ props.
+
+Here are some other resources on the topic if you are interested:
+
+- [README on our composition utilities](https://github.com/microsoft/fluentui/blob/master/packages/react-utilities/src/compose/README.md)
+- [Slots API specification](https://github.com/microsoft/fluentui/blob/master/specs/Slots.md)
+
+---
+
+#### "Ok, that sounds good. Anything else I need to know?"
+
+The FAQ above should cover most of the basics and link to other pieces of documentation for more in-depth reading.
+
+For details about why different decisions were made on specific components refer to the `SPEC.md` file in each component's package.
+
+For specific differences and how to move from a version 8 component to a version 9-BETA component refer to either the `Migration` section within the `SPEC.md` file or to the `MIGRATION.md` file itself in each component's package.
+
+Lastly, if you find something while playing with this new version that you thought should have made it into this FAQ, then please let us know as we very much appreciate your feedback and this is an evolving document.
```

Comment:
```suggestion
Lastly, if you find something while playing with this new version that you thought should have made it into this FAQ, then please let us know through a github issue, we very much appreciate your feedback to our evolving documentation.
```

Should we be a bit clearer how they should get feedback to us ?

Reply:
Agreed! I've made this change